packages/node/test/fixtures/config/config.ts

```diff
@@ -6,7 +6,7 @@ export function buildConfig(config?: Partial<Config>): Config {
   return {
     id: 'my-config',
     triggers: {
-      requests: [{ endpointId: 'endpointId', endpointName: 'endpointName', oisTitle: 'oisTitle' }],
+      request: [{ endpointId: 'endpointId', endpointName: 'endpointName', oisTitle: 'oisTitle' }],
```

Comment:
You missed the example `config.json` files. I think these are only two:

```
packages/node/__dev__/config.json.example
packages/deployer/src/config-data/config.json.example

```

Reply:
Oops